css/dialog.css

```diff
@@ -0,0 +1,75 @@
+.item-dialog {
+    width: 95%;
```

Comment:
Width `95%` is oddly specific.

Reply:
This specific width is for mobile view, that .item-dialog would take up almost the whole width, leaving space to each side. Is there a more correct way?

Reply:
Turns out having width: 100% and margin: auto works fine

Reply:
What if you removed those properties?